service/create/service.go

```diff
@@ -1281,7 +1248,8 @@ func (s *Service) onDelete(obj interface{}) {
 	s.logger.Log("info", fmt.Sprintf("cluster '%s' deleted", cluster.Name))
 }
 
-func (s *Service) onUpdate(oldObj, newObj interface{}) {
+// TODO we need to support this in operatorkit.
+func (s *Service) updateFunc(oldObj, newObj interface{}) {
```

Comment:
We need to support updates in the resource reconciliation framework in case we want to apply it here.